.pre-commit-config.yaml

```diff
@@ -2,33 +2,21 @@ files: ^transform/models/
 
 repos:
   - repo: https://github.com/dbt-checkpoint/dbt-checkpoint
-    rev: v1.1.0
+    rev: v1.1.1
 
     hooks:
       - id: dbt-docs-generate
-        args: ["--cmd-flags", "++project+dir", "transform", "++no+compile"]
+        args: ["--cmd-flags", "++no+compile"]
       - id: check-source-table-has-description
-        args: ["--manifest", "transform/target/manifest.json"]
       - id: check-script-semicolon
-        args: ["--manifest", "transform/target/manifest.json"]
       - id: check-script-has-no-table-name
-        args: ["--manifest", "transform/target/manifest.json"]
       - id: check-script-ref-and-source
-        args: ["--manifest", "transform/target/manifest.json"]
       - id: check-model-has-description
-        args: ["--manifest", "transform/target/manifest.json"]
+        # args: ["--exclude", "^models/demo"] This exclude can be used in case we want to exclude from the get_missing_filepaths logic
```

Comment:
@BAntonellini do we need this?

Reply:
@noel no, we don't. I left it there, commented-out, to keep in mind that in order to use `exclude` it must be passed as part of `args` in certain hooks